zcash_primitives/src/transaction/builder.rs

```diff
@@ -373,6 +367,25 @@ impl<'a, R: RngCore + CryptoRng> Builder<'a, R> {
         }
     }
 
+    /// Creates a new `Builder` targeted for inclusion in the block with the given height
+    /// and randomness source, using default values for general transaction fields.
+    ///
+    /// # Default values
+    ///
+    /// The expiry height will be set to the given height plus the default transaction
+    /// expiry delta (20 blocks).
+    ///
+    /// The fee will be set to the default fee (0.0001 ZEC).
+    pub fn new_with_rng(height: u32, rng: R) -> Builder<'a, R> {
+        let mtx = TransactionData::new();
+        Self::new_with_mtx(height, rng, mtx)
+    }
+
+    pub fn new_with_rng_nu4(height: u32, rng: R) -> Builder<'a, R> {
```

Comment:
There has to be a nicer way to handle this, but we can figure it out in a future refactor of the Rust crates.